### DISCUSSION TITLE
Feat: CodeWhisperer Integration with WarpTerminal.

### DISCUSSION THREAD

**Discussion by ggiallo28** [aws/codewhisperer-command-line-discussions#3] [Feature request]
### Checks

- [X] I could not find an existing setting that solved my issue
- [X] I searched [the discussions in this repo](https://github.com/aws/codewhisperer-command-line-discussions/discussions?q=) and there are no duplicates of my feature request

### Description:
> Please include a detailed description of the feature (and an image or screen recording, if applicable)

Currently CodeWhisperer does not support Warp terminal.

**Comment by grant0417** [comment]
Will link to the [old Fig warp support issue](https://github.com/withfig/fig/issues/785), but basically it isnt possible for us to integrate with Warp due to them being a closed source terminal.

**Comment by katzueno** [comment]
Same here.. but understandand that Warp is closed source and hard to ask.
Perhaps, we should all send the reuquest to Warp then.
